Flow.Launcher.Core/ExternalPlugins/PluginsManifest.cs

```diff
@@ -10,43 +11,45 @@ namespace Flow.Launcher.Core.ExternalPlugins
 {
     public static class PluginsManifest
     {
-        static PluginsManifest()
-        {
-            UpdateTask = UpdateManifestAsync();
-        }
-
-        public static List<UserPlugin> UserPlugins { get; private set; } = new List<UserPlugin>();
+        private const string manifestFileUrl = "https://raw.githubusercontent.com/Flow-Launcher/Flow.Launcher.PluginsManifest/plugin_api_v2/plugins.json";
```

Comment:
By the way, could you change this to jsdelivr link to help Chinese users? Not sure whether they support etag as well.

Reply:
done, it's significantly faster as well, github takes ~50ms to return a 304 for me, jsdelivr takes ~20ms